### ISSUE TITLE
Debug symbols in CRAN builds

### ISSUE THREAD

**Issue by hannes** [duckdb/duckdb-r#93]
```
curl -I https://cran.r-project.org/bin/macosx/big-sur-arm64/contrib/4.3/duckdb_0.9.2-1.tgz 
...
Content-Length: 85023346
```

The file is over 80 MB. When unpacked, it contains ca. 300 MB of debug symbols (`duckdb.so.dSYM`). 

We should investigate whether it is possible to not include those in the download.

CC @hadley

**Comment by hadley** [comment]
@gaborcsardi any ideas?

**Comment by gaborcsardi** [comment]
Yeah, unfortunately that is on purpose, see e.g. https://stat.ethz.ch/pipermail/r-sig-mac/2022-June/014500.html

**Comment by gaborcsardi** [comment]
It is only an issue on macOS, Windows and (PPM) Linux binaries are around or under 9MB.

**Comment by hannes** [comment]
will reach out to r-sig-mac

**Comment by hannes** [comment]
Post is here: https://stat.ethz.ch/pipermail/r-sig-mac/2024-April/014960.html

**Comment by krlmlr** [comment]
We're good here now.
